sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/RestOperations/AzureReservationAPIRestOperations.cs

```diff
@@ -42,13 +42,14 @@ internal HttpMessage CreateGetCatalogRequest(string subscriptionId, string reser
         {
             var message = _pipeline.CreateMessage();
             var request = message.Request;
+            var apiVersion = reservedResourceType.Equals("SqlDatabases", StringComparison.OrdinalIgnoreCase) ? "2020-10-01-preview" : _apiVersion;
```

Comment:
SqlDatabases is currently not supported in 2022-03-01 version, using 2020-10-01-preview for it instead